source/definitions.yaml

```diff
@@ -105,6 +105,14 @@ definitions:
       creator_id:
         type: string
 
+  ChannelStats:
+    type: object
+    properties:
+      id:
```

Comment:
I prefer to be more consistent here maybe as with ChannelMember where `id` is actually `channel_id`?